src/helpers/FixOpenApiSpec/Program.cs

```diff
@@ -165,4 +154,121 @@ static OpenApiSchema FromJson([StringSyntax(StringSyntaxAttribute.Json)] string
     }
     
     return schema;
-}
\ No newline at end of file
+}
+
+// Add response schema for deployments create/update
+openApiDocument.Paths["/deployments"].Operations[OperationType.Post].Responses["200"] = new OpenApiResponse
```

Comment:
@aider

It's all should be before `jsonOrYaml = openApiDocument.SerializeAsYaml(OpenApiSpecVersion.OpenApi3_0);`